orttraining/orttraining/test/python/orttraining_on_device_training_tests.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import logging
+import os
+import sys
+
+from _test_commons import run_subprocess
+
+logging.basicConfig(format="%(asctime)s %(name)s [%(levelname)s] - %(message)s", level=logging.DEBUG)
+log = logging.getLogger("OnDeviceTrainingTests")
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cwd", help="Path to the current working directory")
+    return parser.parse_args()
+
+
+def run_onnxblock_tests(cwd, log):
+    log.debug("Running: onnxblock tests")
+
+    command = [sys.executable, "-m", "pytest", "-sv", "orttraining_test_onnxblock.py"]
+
+    run_subprocess(command, cwd=cwd, log=log).check_returncode()
+
+
+def run_onnxruntime_test_all_ctest(cwd, log, filter):
+    command = [os.path.join(cwd, "onnxruntime_test_all"), f"--gtest_filter={filter}"]
+
+    run_subprocess(command, cwd=cwd, log=log).check_returncode()
+
+
+def run_training_api_tests(cwd, log):
```

Comment:
I think we don't need run this explicitly from there. Those tests will be included in unit test run once our build flag is enabled.